#!/usr/bin/env python
PACKAGE = "hsvhfb"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()



gen.add("select",    bool_t,   0, "pixel_selection",                          False)
gen.add("hue",       double_t, 0, "Hue",                                      .00, 0, 1)
gen.add("tolerance", double_t, 0, "Hue tolerance",                            .10, 0, 1)
gen.add("min_sat",   double_t, 0, "Minimal saturation required",              .10, 0, 1)
gen.add("min_val",   double_t, 0, "Minimal intensity value required",         .10, 0, 1)


exit(gen.generate(PACKAGE, "hsvhfb", "SelectionParam"))
